packages/peregrine/package.json

```diff
@@ -17,6 +17,7 @@
   },
   "homepage": "https://github.com/magento/pwa-studio/tree/master/packages/peregrine#readme",
   "devDependencies": {
+    "@testing-library/react-hooks": "~1.1.0",
```

Comment:
Sorry, but I'm not interested in adding a new testing library to PWA Studio at this time. I understand that it may be a somewhat popular library, but that's not part of the acceptance criteria here.

In general, please avoid adding new dependencies that haven't been pre-approved.